posts/2021/09/re-engineering-cicd-pipelines-for-scipy.md

```diff
@@ -0,0 +1,209 @@
+
+## Future Plans
+
+There is a lot to do with the new CI matrix and the configurations, and it will take more time to reach the desired maturity. I was able to meet most of the goals, but some of them remain.
+
+- Implement the CI matrix on the GitHub Actions workflow.
+
+- Add associated developer documentation for CI matrix and service providers.
+
+- Improve and optimize the project as it is tested on multiple architectures.
+
+- Automate source and binary distribution through GitHub releases.
+
+## What did I learn from this project?
+
+The summer internship at Quansight Labs has been the busiest time of my life for all good reasons. I learned a great deal about GitHub Actions, build systems, and PyData projects through the same. I also learned about testing and benchmarking, along with Docker by a significant margin. I had the basic idea for some of them and have worked on them in hobby projects, but implementing them myself on a significant project and fixing bugs was a wholesome experience.
+
+Apart from the same, I learned a great deal about the communication side of things. Communicating with my mentor and the stakeholders improved my thought process and intuition behind how things work. It also helped me instill a “doer” mentality, where I work fast, fail fast, and iterate quickly. Getting to know fellow Interns and the great work they have been doing was also significant for me to understand the ecosystem.
+
+## Acknowledgements
+
+My internship at Quansight Labs has been one of the most wholesome experiences I ever had. I’m grateful to have worked on this project and get acquainted with Open Source, being in my undergraduate studies itself. Open Source has taught me a great deal about many things, and this project has further paved the way for me to cherish my ambitions of working more on the Python and DevOps side of things.
+
+I want to thank my mentor, John Lee, for his impeccable mentorship and feedback. Without his help and support, all this would not have been possible. Above all, his skill in drawing from past experiences to future model outcomes has been helpful for me to understand things beyond open source and software development itself.
+
+I would also like to thank Ralf Gommers, who has been instrumental in my project work. His feedback, continued support, and long-term vision contributed significantly to my project. [Anirudh Dagar](https://github.com/AnirudhDagar/), my co-intern, helped me understand more about SciPy and helping with my first PR reviews. Jaime helped me considerably understand benchmarking, and getting to know more about his work inspired me. They all are very polite, knowledgeable, and helpful.
```

Comment:
You're using the full name for everyone here except Jaime - I suggest adding his last name too.

Reply:
I have used his full name before so I have reverted to use the first name for everyone in the `Acknowledgements` section.